### PR TITLE
style.css: cursor pointer added to navbar tabs

### PR DESCRIPTION
**Pull request by gabru-md**
Adds the Pointer type cursor to Navbar Tabs so that user know that
they are clickable.

Closes https://github.com/coala/projects/issues/318

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: resources/css/style.css [hunk 1]
```diff
@@ -11,6 +11,9 @@
 .report .link {
   padding: 0.5em;
 }
+.tab{
```
Comment (adtac) [inline resources/css/style.css]: space betweeen "tab" and "{" please

**Comment by adtac** [comment]
unack https://github.com/coala/projects/pull/373/commits/5c0c3a5c862296ed2cd12edee3e136039fbbba40

**Comment by gabru-md** [comment]
@adtac Can you assign this to me. I'll make the changes but this PR is not assigned to me anyhow.
I asked for it on gitter channel as well.

**Comment by gabru-md** [comment]
@adtac  please review

**Comment by gabru-md** [comment]
@nvzard @adtac please review now!
Thanks!

**Comment by jayvdb** [comment]
Commit message:

> style.css: cursor pointer added to navbar tabs

should be

> style.css: Cursor pointer added to navbar tabs

**Comment by gabru-md** [comment]
Will do it.

**Comment by gabru-md** [comment]
@jayvdb @adtac Please review

**Comment by jayvdb** [comment]
unack https://github.com/coala/projects/pull/373/commits/c61d5881d8bb6826d8ba52cec5f5ec8f62f87b89

**Comment by jayvdb** [comment]
Please read the commit guidelines carefully.

**Comment by RaiVaibhav** [comment]
@gabru-md  Ping as pointed by @aashutoshrathi  `GCI` is close so please change it asap as your changes worked fine but need a little work in commit message put `space` between `:` and  `Cursor`

**Comment by RaiVaibhav** [comment]
If you need further help please got [through](https://coala.io/newcomer) you will get all the details here

**Comment by gabru-md** [comment]
@RaiVaibhav I'll close this PR and open from a new branch. :+1:

**Pull request closed without merging**